game/txt/Makefile

```diff
@@ -17,9 +17,15 @@ TXT=help.txt news.txt
 # below.)
 INDEX_FLAGS=
 
+CHANGES_FILES=$(wildcard ../../CHANGES*)
+HELP_CHANGES_FILES=$(patsubst ../../CHANGES.%,hlp/pennv%.hlp,$(CHANGES_FILES))
+
 all: $(TXT)
 
-help.txt: hlp/*.hlp hlp compose.sh
+$(HELP_CHANGES_FILES): hlp/pennv%.hlp: ../../CHANGES.%
+	../../utils/mkvershlp.pl hlp ../../CHANGES*
```

Comment:
@talvo Given that our [Windows instructions](https://github.com/pennmush/pennmush/wiki/InstallingWindows) already require ActivePerl, we could put a .bat file in win32 to run the script?